jujugui/static/gui/src/app/components/help/_help.scss

```diff
@@ -0,0 +1,205 @@
+
+.help {
+  $help-max-width: 850px;
+
+  &.lightbox {
+    @media screen and (min-width: $help-max-width) {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+  }
+
+  .lightbox__close {
+    @media screen and (max-width: $help-max-width - 1) {
+      top: .5rem;
+      right: .5rem;
+    }
+  }
+
+  .lightbox__wrapper {
+    text-align: left;
+
+    @media screen and (min-width: $help-max-width) {
+      width: 850px;
+      left: auto;
+      right: auto;
+      top: auto;
+      bottom: auto;
+    }
+
+    @media screen and (max-width: $help-max-width - 1) {
+      top: 2.5rem;
+      left: 0;
+      right: 0;
+      bottom: 0;
+    }
+
+    .lightbox__content {
+      padding: 0;
+
+      @media screen and (min-width: $help-max-width) {
+        height: 530px;
+        overflow: hidden;
+      }
+
+      @media screen and (max-width: $help-max-width - 1) {
+        overflow: auto;
+      }
+    }
+  }
+
+  &__header {
+    padding: 1.5rem;
+
+    &-title {
+      float: left;
+      margin: 0;
+      line-height: 40px;
+    }
+
+    &-search {
+      float: right;
+      padding: 0;
+    }
+  }
+
+  &__content {
+    padding: 0 1.5rem 1.5rem;
+    display: flex;
+
+    @media screen and (max-width: $breakpoint-medium) {
+      flex-direction: column;
+
+      .p-card:nth-child(2) {
+        order: 1;
+      }
+    }
+
+    .p-card {
+      width: 100%;
+      margin-left: 1.5rem;
+
+      &:nth-child(2),
+      &:nth-child(3) {
+        background: $color-light;
+      }
+
+      &:first-child {
+        margin-left: 0;
+      }
+
+      @media screen and (max-width: $breakpoint-medium) {
+        margin: {
+          left: 0;
+          top: 1.5rem;
+        }
+
+        &:first-child {
+          margin-top: 0;
+        }
+      }
+    }
+
+    .link {
+      font-weight: 300;
+      color: $link-blue;
+      cursor: pointer;
+    }
+  }
+
+  &__footer {
+    display: block;
+    background: $dark-header;
+    padding: 1.5rem;
+    border-radius: 0 0 $border-radius $border-radius;
+    color: $white;
+
+    a, .link {
+      color: $white;
+      cursor: pointer;
+
+      &:hover {
+        text-decoration: underline;
+        color: $white;
+      }
+    }
+
+    .external {
+      background-image: $svg-external;
+    }
+
+    @media screen and (max-width: $breakpoint-medium) {
+      display: flex;
```

Comment:
is flex the right tool for the job here? Seems like it's a grid shape...

Reply:
It is until you get below $breakpoint-medium - mobile devices. At that point, they become a column.
CSS Grid is great, but overkill for some things.